components/feature/prompts/src/main/java/mozilla/components/feature/prompts/provider/FileProvider.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package mozilla.components.feature.prompts.provider
+
+typealias AndroidxFileProvider = androidx.core.content.FileProvider
```

Comment:
Since we only use this value once, I think it should be used directly instead of adding a type alias.

```
class FileProvider : androidx.core.content.FileProvider() 
```

Also, the file should have a newline at the end.

Reply:
👍